tests/admin_checks/tests.py

```diff
@@ -124,6 +124,29 @@ def test_custom_get_form_with_fieldsets(self):
         errors = ValidFormFieldsets.check(model=Song)
         self.assertEqual(errors, [])
 
+    def test_fieldsets_tuple(self):
+        """
+        Tests for basic tuple/lists within field sets (#24089)
+        """
+        class NotATupleAdmin(admin.ModelAdmin):
+            list_display = ["pk", "title"]
+            list_editable = ["title"]
+            fieldsets = [
+                (None, {
+                    "fields": ("title")  # expect a comma
+                }),
+            ]
+        errors = NotATupleAdmin.check(model=Song)
+        expected = [
+            checks.Error(
+                "The value of 'fieldsets[0]' must be a list or tuple.",
```

Comment:
If I'm reading the code correctly, `fieldsets[0]` is a tuple. Perhaps the error message should be `"The value of 'fieldsets[1]['fields']' must be a list or tuple."`